easybuild/easyblocks/c/clang.py

```diff
@@ -215,21 +222,27 @@ def configure_step(self):
         self.cfg.update('configopts', "-DGCC_INSTALL_PREFIX='%s'" % gcc_prefix)
         self.log.debug("Using %s as GCC_INSTALL_PREFIX", gcc_prefix)
 
+        # Configure some default options
         self.cfg.update('configopts', "-DCMAKE_BUILD_TYPE=Release")
+        self.cfg.update('configopts', '-DLLVM_REQUIRES_RTTI=ON')
+        self.cfg.update('configopts', '-DLLVM_ENABLE_RTTI=ON')
+        self.cfg.update('configopts', '-DLLVM_ENABLE_EH=ON')
+        self.cfg.update('configopts', '-DCLANG_DEFAULT_OPENMP_RUNTIME=libomp')
```

Comment:
@ocaisa What if someone wants to build with different options?

Also, do all of these make sense for the oldest `Clang` versions we support?

Reply:
Answer to that is I have no idea, I don't know what they do, I just know Slack have them set. I can just wrap these options in a version check (for Clang 8)?